app/src/main/java/com/internship/move/OnboardingFragment.kt

```diff
@@ -0,0 +1,33 @@
+package com.internship.move
```

Comment:
Use a main `feature` package inside `move` and nested packages for each feature's files
![image](https://user-images.githubusercontent.com/84084708/184794000-db710e4c-ce25-44cf-bdce-a3630712105d.png)